### PR TITLE
Db updates

### PR DESCRIPTION
**Pull request by irfamustafa**
Updated the DB Schema and corresponding insert statements by removing the type table and just placing a category (i.e. type) column into modules table

### CODE REVIEW

**Review by apj11c — DISMISSED** [review]

File: DBInsertData.sql [hunk 1]
```diff
@@ -8,14 +8,14 @@ Insert INTO Class VALUES ('JV123', 'FSD123', 'IN5');
 
 Insert INTO Employees VALUES('IM505', 'IceMan', 'IM505@syn.com', 'JV123');
 
-Insert INTO Type VALUES('Java', 'FSD123');
-Insert INTO Type VALUES('DB', 'DB343');
+--Insert INTO Type VALUES('Java', 'FSD123');
```
Comment (kabollinger03) [inline DBInsertData.sql]: These don't need to be commented out, because the script is going to be run by someone who has never run it before.

**Review by kabollinger03 — CHANGES_REQUESTED** [review]

**Review by ChrisSMendoza — DISMISSED** [review]

**Comment by Lyx-22** [comment]
I think there should be a isValidated column in Users to make sure the the email address (username) is correct. We should not store passwords inside the DB for security reasons and use a hashed value instead. I am not sure if should have an extra column in Users for ConfirmPassword (When I learned Ruby on Rails, the table would have this extra column). This extra Password field would be used when changing passwords.

**Review by kjones2296 — DISMISSED** [review]

File: DBSchemaTables.sql [hunk 2]
```diff
@@ -71,12 +71,13 @@ CREATE TABLE Student_Performance.Type (
 CREATE TABLE Student_Performance.Modules (
   module_id NUMBER(10) NOT NULL,
   module_name VARCHAR2(45) NULL,
-  module_type VARCHAR2(30) NOT NULL,
+  category VARCHAR2(30) NULL,
```
Comment (kabollinger03) [inline DBSchemaTables.sql:75]: Category should be not null

**Comment by kabollinger03** [comment]
users need username to be varcher2(100)
users password should probably be varchar2(30)
class doesn't need a column for user_id
employees name should go up to 90 or 100 to mirror change in users.username
employees email should go up to 100, also to mirror this change 
courses may need a big VARCHAR for description, but that is totally optional for now
Why do we need the table Class_has_Modules? For one, it is stream_has_modules and class_has_stream. For two, class has a module_id column, and modules has a stream_id column,

File: DBSchemaTables.sql [hunk 3]
```diff
@@ -25,7 +26,7 @@ CREATE TABLE Student_Performance.Stream (
 CREATE TABLE Student_Performance.Class (
   class_id VARCHAR2(30) NOT NULL,
   stream_id VARCHAR2(40) NOT NULL,
-  user_id VARCHAR2(30) NOT NULL,
+  user_id VARCHAR2(100) NOT NULL,
```
Comment (kabollinger03) [inline DBSchemaTables.sql:29]: Doesn't /Need/ a user_id but there is no harm in keeping it.

**Review by kabollinger03 — APPROVED** [review]
Minor comments, not hurting anything really.

The type table (type-stream I think is the purpose) can go away but it isn't hurting anything. Streams don't have types, modules do. And module type is now a column.

Courses could possibly have a description column, but I would not worry too much about that.

**Review by kjones2296 — APPROVED** [review]

**Comment by kjones2296** [comment]
Kate and Kevin approved changes to database

**Pull request merged**